### ISSUE TITLE
[Adaptive Card > ExpenseReport.Json]: 'ER-13052' link under Expense Report card is not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#177]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "ExpenceReport.json" control and double tap to activate it.
7. Perform right swipe and navigate to the card and observe the issue.

**Actual Result:**
ER_13052 link under expense report card is not accessible through the swipe gesture.

**Observations:**
Same issue is repro through the keyboard as well.

**Expected Result:**
'ER-13052' link should be accessible through the swipe gesture.

**User Experience:**
It will impact the screen reader user as they will not able to access the control due to this they will not able to get the information contain under it.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/keyboard

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/c02aedd9-788c-4a43-9a0f-f7025ca753c5

### Card JSON

```JSON
NA
```

**Comment by GabrielMedAlv** [comment]
This has been fixed on https://github.com/microsoft/Teams-AdaptiveCards-Mobile/pull/249

**Comment by vagpt** [comment]
This issue is now working fine, hence closing this issue.

Build - **1.0 (2.4.0-beta.24.11.3.1**

![Image](https://github.com/user-attachments/assets/6c969829-ac15-4cd5-9000-56195211b3d7)
